problems/1.intro.js

```diff
@@ -3,8 +3,9 @@
  * 
  * @returns undefined
  */
-function helloWorld() {
-
+function helloWorld(){
+  console.log ("hello world")
+  return undefined;
```

Comment:
No need to explicitly return undefined. A function with no return will automatically return undefined.